web/docs/guides/api.mdx

```diff
@@ -197,51 +210,86 @@ let { data: todos, error } = await supabase
 
 ```bash
 # Append /rest/v1/ to your URL, and then use the table name as the route
-curl '[SUPABASE_URL]/rest/v1/todos' \
--H "apikey: [SUPABASE_ANON_KEY]" \
--H "Authorization: Bearer SUPABASE_ANON_KEY"
+curl '<SUPABASE_URL>/rest/v1/todos' \
+-H "apikey: <SUPABASE_ANON_KEY>" \
+-H "Authorization: Bearer <SUPABASE_ANON_KEY>"
 ```
 
 </TabItem>
 </Tabs>
 
-#### Details and links:
-
-- When you create a table in Postgres, Row Level Security is disabled by default. Make sure you secure it by [enabling RLS](/docs/guides/api#securing-your-routes).
-- Never expose the `service_role` key in a browser or anywhere where a user can see it.
-- JS Reference: [select()](/docs/reference/javascript/select), 
+JS Reference: [select()](/docs/reference/javascript/select), 
 [insert()](/docs/reference/javascript/insert),
 [update()](/docs/reference/javascript/update),
 [upsert()](/docs/reference/javascript/upsert),
 [delete()](/docs/reference/javascript/delete),
 [rpc()](/docs/reference/javascript/rpc) (call Postgres functions).
 
-### Safeguards towards accidental deletes and updates
 
-For new projects, by default, the Postgres extension [safeupdate](https://github.com/eradman/pg-safeupdate) is enabled for all queries coming from the API. 
-This ensures that any `delete()` or `update()` would fail if there are no accompanying filters provided.
-This can be disabled by running the following query on the SQL editor on the dashboard:
+### GraphQL API 
 
-```sql
-alter role "authenticator"
-set "session_preload_libraries" to default;
+You can use any GraphQL client with the Supabase GraphQL API. For our GraphQL example we will use [urlql](https://formidable.com/open-source/urql/docs/). 
+
+<Tabs
+defaultValue="Javascript"
+values={[
+  {label: 'Javascript', value: 'Javascript'},
+  {label: 'cURL', value: 'cURL'},
+]}>
+<TabItem value="Javascript">
+
+```javascript
+import { createClient, useQuery } from 'urql';
+
+const client = createClient({
+  url: '<SUPABASE_URL>/grapqhl/v1',
+})
+
+const TodosQuery = `
+  query {
+    todos {
+      id
+      title
+    }
+  }
+`
+
+const [result, reexecuteQuery] = useQuery({
+  query: TodosQuery,
+})
+
+const { data, fetching, error } = result
 ```
 
-To reenable it :
-```sql
-alter role "authenticator" 
-set "session_preload_libraries" = 'safeupdate';
+</TabItem>
+<TabItem value="cURL">
+
+```bash
+# Append /grapqhl/v1/ to your URL, and then use the table name as the route
+curl --request POST '<SUPABASE_URL>/grapqhl/v1' \
+-H 'apikey: <SUPABASE_ANON_KEY>' \
+-H 'Authorization: Bearer <SUPABASE_ANON_KEY>' \
+-d '{ "query":"{ todos(first: 3) { edges { node { id } } } }" }'
 ```
-The above can also be run to enable it for existing projects. Settings are applied immediately and as such no restarts are needed after running any of the two.
 
-### Managing Realtime 
+</TabItem>
+</Tabs>
+
```

Comment:
@olirice - can the users access graphiql for this endpoint?

Reply:
no, we don't serve GraphIQL at the endpoint

The `apiKey` makes it difficult to make it navigable from the browser without also leaking the key

Reply:
OK no probs - something to revisit later

Reply:
No, but the example app showcases how one might set GraphiQL up.

Reply:
I don't have context on how hard this would be but this is the snippet we'd need to serve to enable it

```html
<html>
  <head>
    <title>GraphiQL</title>
    <link href="https://unpkg.com/graphiql/graphiql.css" rel="stylesheet" />
  </head>
  <body style="margin: 0;">
    <div id="graphiql" style="height: 100vh;"></div>

    <script
      crossorigin
      src="https://unpkg.com/react/umd/react.production.min.js"
    ></script>
    <script
      crossorigin
      src="https://unpkg.com/react-dom/umd/react-dom.production.min.js"
    ></script>
    <script
      crossorigin
      src="https://unpkg.com/graphiql/graphiql.js"
    ></script>

    <script>
      const fetcher = GraphiQL.createFetcher({
              url: 'https://<project_ref>.supabase.co/graphql/v1',
             headers: {
                 "apiKey": "<apiKey>"
             }
      });
      ReactDOM.render(
        React.createElement(GraphiQL, { fetcher: fetcher }),
        document.getElementById('graphiql'),
      );
    </script>
  </body>
</html>
```

Reply:
Personally, I'd leave GraphiQL up to a developer to implement  and instead use Paw or Insomnia to use as an IDE/Browser.

* https://paw.cloud
* https://insomnia.rest

Reply:
damn, this is cool @olirice - I will see if I can get this into the docs somehow